src/engine/CountAvailablePredicates.cpp

```diff
@@ -220,8 +230,8 @@ void CountAvailablePredicates::computePatternTrick(
     const vector<A>* input, vector<array<Id, 2>>* result,
     const vector<PatternID>& hasPattern,
     const CompactStringVector<Id, Id>& hasPredicate,
-    const CompactStringVector<size_t, Id>& patterns,
-    const size_t subjectColumn) {
+    const CompactStringVector<size_t, Id>& patterns, const size_t subjectColumn,
+    RuntimeInformation& runtimeInfo) {
```

Comment:
Hmm, the Google style guide actually forbids non-const references unless an API needs them, however we haven't really enforced that and they are scattered around the code base.

Reply:
I've changed the parameter to be a pointer instead.